student.rb

```diff
@@ -1,9 +1,12 @@
 require_relative 'Person'
 
 class Student < Person
+  attr_accessor :classroom
+
   def initialize(age, classroom, name = 'Unknown', parent_permission: true)
     super(name, parent_permission, age)
     @classroom = classroom
+    classroom.students << self
```

Comment:
- [ ] There should be a method that serves as the 'belongs-to' side (a student belongs to a classroom), it should be responsible for adding the student if it is not there before. More like this:

```
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```